packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/studio/services/EditingContextInitializer.java

```diff
@@ -121,6 +121,6 @@ public void preProcess(IEditingContext editingContext) {
 
     private void toResource(String editingContextId, ResourceSet resourceSet, Document document) {
         boolean useMigrationParticipants = this.migrationParticipantPredicates.stream().anyMatch(predicate -> predicate.test(editingContextId));
-        this.resourceLoader.toResource(resourceSet, document.getId().toString(), document.getName(), document.getContent(), useMigrationParticipants);
+        this.resourceLoader.toResource(resourceSet, document.getId().toString(), document.getName(), document.getContent(), useMigrationParticipants, true);
```

Comment:
It doesn't really matter if it is `true` or `false` here, the resource won't be persisted anyways